_posts/2017-11-29-daniel-test.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+title: Hello World!
+---
+
+Hello World! Remove me!!
```

Comment:
Glad you were testing things out. Please remove this file from your PR branch.

Reply:
whoops. that's embarrassing. I removed the file from my branch.